pkg/storage/cassandra_dependencies.go

```diff
@@ -42,6 +42,12 @@ func cassandraDeps(jaeger *v1.Jaeger) []batchv1.Job {
 		host = "cassandra" // this is the default in the image
 	}
 
+	keyspace := jaeger.Spec.Storage.Options.Map()["cassandra.keyspace"]
+	if keyspace == "" {
+		jaeger.Logger().Info("Cassandra keyspace not specified. Using 'jaeger_v1_test' for the cassandra-create-schema job.")
+		keyspace = "jaeger_v1_test" // this is default in the image
```

Comment:
Keyspace should contain datacenter if it is specified 

https://github.com/jaegertracing/jaeger/blob/b2a159a67b3d937eb29fe2a98f5b05708d34f56a/plugin/storage/cassandra/schema/create.sh#L44

Reply:
Maybe we can rely on the script to provide the defaults. Have a look at `util.RemoveEmptyVars` function.

Reply:
If we look at Jaeger from other way. One cassandra can be used by different application based on `KEYSPACE`. So, user should be given flexibility to define it.

Reply:
You are right, I am fine allowing the full control on the keyspace assuming it can be fully specified.